plugins/terraform-backend/src/service/router.ts

```diff
@@ -28,7 +28,16 @@ export async function createRouter(
       findWorkspace(token, organization, workspaceName)
         .then(workspace => getRunsForWorkspace(token, workspace.id))
         .then(runs => {
-          response.json(runs);
+          response.json(
```

Comment:
Why do we need this change? `getRunsForWorkspace` should return us the data in the format required

Reply:
Typecheck failed as it doesn't recognise the type, so this is just a workaround. If you have any cleaner suggestions, they are welcome.